pacman-helper.sh

```diff
@@ -273,14 +317,23 @@ quick_add () { # <file>...
 			die "Could not initialize $dir/$arch"
 		fi
 
+		case "$label" in
+		remove)
+			test -z "$GPGKEY" ||
```

Comment:
I think this would be better
```suggestion
			test -f "$arch/$file.sig" &&
```

but it doesn't really matter, the only time there's not a GPGKEY should be when it's a dry run, so it'll only change output of what actions it would have done.

Reply:
That would not actually work because we're in a sparse checkout; Only the package database is present at this stage.